static/sass/styles.scss

```diff
@@ -8,15 +8,19 @@ $color-tabs-active-bar: #e95420;
 $color-brand: #7c355d;
 $color-dark: #262626;
 $color-paper: #f3f3f3;
+
+//XXX: move this to vanilla
```

Comment:
Can you add a link to the issue in vanilla to track this?

Reply:
will do

Reply:
done